src/Service/MoreInfoService/AbstractMoreInfoService.php

```diff
@@ -83,8 +83,8 @@ abstract class AbstractMoreInfoService extends SoapClient
      *
      * @param Type $index
      *   Elastica index
-     * @param LoggerInterface $statsLogger
-     *   Statistics logger
+     * @param StatsLoggingService $statsLoggingService
```

Comment:
The logger service implements the `LoggerInterface` this should be the type that the function takes.

Reply:
But this is autowired. Therefore it needs to be the precise service.